src/Entity/EntityScopeVisibilityTrait.php

```diff
@@ -22,7 +22,8 @@ trait EntityScopeVisibilityTrait
      *     "pap_campaign_read",
      *     "pap_campaign_read_after_write",
      *     "phoning_campaign_read",
-     *     "phoning_campaign_list"
+     *     "phoning_campaign_list",
+     *     "jecoute_news_read_dc"
```

Comment:
You can add `,` in the end, and if we add a new group we will see only one line as modified.
```suggestion
     *     "jecoute_news_read_dc",
```